simpleclient_common/src/main/java/io/prometheus/client/exporter/common/TextFormat.java

```diff
@@ -46,6 +46,10 @@ public static void write004(Writer writer, Enumeration<Collector.MetricFamilySam
         }
         writer.write(' ');
         writer.write(Collector.doubleToGoString(sample.value));
+        if (sample.timestamp != null){
+        	writer.write(' ');
```

Comment:
could you remove the tab and use spaces for indents?

Reply:
Sorry, I didn't realize about it, I have changed it, but still not pushed.